packages/host/app/lib/current-run.ts

```diff
@@ -278,8 +278,16 @@ export class CurrentRun {
 
       let { content, lastModified } = fileRef;
       if (url.href.endsWith('.json')) {
-        let { data: resource } = JSON.parse(content);
-        if (isCardResource(resource)) {
+        let resource;
+
+        try {
+          let { data } = JSON.parse(content);
+          resource = data;
+        } catch (e) {
+          log.debug(`unable to parse ${url.href} as card JSON`);
```

Comment:
This is a guess at how this should be handled, does something else need to happen if the parsing fails?

Reply:
I guess it would be good to actually tell the user? Maybe in some kind of an error display

Reply:
I’ve changed it to `warn`, this is something that happens with realm server indexing so it’s not directly user-facing. A related problem is that broken JSON silently (console-only) fails to save, the only way it’ll save is if it’s valid:

<img width="644" alt="Boxel 2023-09-28 10-50-09" src="https://github.com/cardstack/boxel/assets/43280/cb288826-3cb6-4f60-bd8a-566ac5250a19">